Source/Order/Orders.h

```diff
@@ -0,0 +1,159 @@
+/**
+ * Class header for the Order class and its subclasses
+ * Deploy, Advance, Bomb, Blockade, Airlift, and Negotiate.
+ * The class header also has the OrdersList class.
+ * 
+ * @author Mark Nasol
+ */
+#pragma once
+
+#include <string>
+#include <list>
+using namespace std;
+
+/**
+ * The function and member declarations of the Order class 
+ */
+class Order
+{
+public:
+    Order();                                             //Constructor
+    ~Order();                                            //Destructor
+    Order(const Order &od);                              //Copy constructor
+    Order &operator=(const Order &od);                   //Assignment operator
+    friend ostream &operator<<(ostream &out, Order *od); //Stream insertion operator
+
+    Order(string typeValue, string descValue);
+    bool validate();
+    virtual void execute();
+    string getType();
+    string getDescription();
+    int getTerritory();
+    int getNumArmies();
+    void setType(string newType);
```

Comment:
I am removing the type setter for the Order class since we are initializing an Order's type only once and there is no need for further modification of the order type